### DISCUSSION TITLE
IDE

### DISCUSSION THREAD

**Discussion by namrata18s** [TreyWW/MyFinances#121] [Q&A] [answered]
In the setup.md file the steps are given for Pycharm PRO IDE. Is it preferred to use pycharm or we can use use IDE of our choice? can someone please clarify on this?

**Comment by TreyWW** [comment, marked as answer]
Hi there @namrata18s. Yeah so in the setup I mostly showed Pycharm as their Django support is amazing in my opinion. Quite a few of us use it too. 

But you don't need to use it, if you don't want to use it you can follow setup and just follow all of the opposite steps. So you'll likely:
- setup your .env file manually
- setup a database (of choice or you can use XAMPP to create one - in the guide)
- Add database details to your .env
- run `py manage.py runserver` to run the app. 

We have recently added tailwind via NPM so it's suggested that you use `npm install` and then `npm run tailwind-build`

Feel free to ask any more questions as needed, I appreciate your interest in the project.

**Reply by TreyWW** [reply]
I've now added a Wiki Section for this, hopefully it's a bit clearer.

https://github.com/TreyWW/MyFinances/wiki/Setting-up-Django

Thanks for the question though, hopefully the new guide is a bit easier to follow
